protocols/dcutr/src/behaviour_impl.rs

```diff
@@ -386,3 +460,7 @@ impl NetworkBehaviour for Behaviour {
         }
     }
 }
+
+fn is_relayed(addr: &Multiaddr) -> bool {
+    addr.iter().any(|p| p == Protocol::P2pCircuit)
+}
```

Comment:
No extension trait? Not a strong opinion.

Reply:
Didn't think it was worth it if the usage is only in a single file.